internal/deviations/deviations.go

```diff
@@ -645,4 +655,8 @@ var (
 	routePolicyUnderAFIUnsupported = flag.Bool("deviation_route_policy_under_afi_unsupported", false, "Set true for device that does not support route-policy under AFI/SAFI, default is false")
 
 	interfaceRefConfigUnsupported = flag.Bool("deviation_interface_ref_config_unsupported", false, "Device does not support interface-ref configuration when applying features to interface")
+
+	platformPowerEnableWaitRequired = flag.Int("deviation_platform_power_enable_wait_required", 0, "Set timer in minutes to wait for shut platform to power up")
```

Comment:
With the new method of adding deviations, you no longer have to add these flag values here. So these lines can be removed.